network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,137 @@
+
+### Choosing a Verify Certificate setting
+
+If you select the HTTPS protocol, you are prompted to choose a Verify Certificate setting. Note that this setting does not display if you selected HTTP or TCP protocol.
+
+When the Load Balancer initiates an HTTPS connection to a backend server, the backend server presents its certificate. 
+- When Verify Certificate is selected, the Load Balancer checks the backend server's certificate and only connects if it is valid.
+- When Verify Certificate is unselected, the Load Balancer does not check the backend server's certificate, connecting to the backend server regardless of the certificate's validity. 
+
+This setting mostly concerns those using self-signed certificates on the backend server/s.
+
+Note that the same Verify Certificate setting you select here will also be used for health checks, if you choose HTTPS for your [health check method](#choosing-a-health-check-method).
+
+### Choosing a proxy protocol
+
+Proxy protocol determines how connection information from the client (eg the client’s IP address) will be transferred through the Load Balancer and on to the destination backend server. This allows the backend server to know the client's address, whatever the upper layer protocol (HTTP/HTTPS/TCP). 
+
+* **No proxy protocol**:  Disable proxy protocol.
+* **Proxy protocol v1**: Version one (text format).
+* **Proxy protocol v2**: Version two (binary format).
+* **Proxy protocol v2-ssl**: Version two with SSL connection
+* **Proxy protocol v2-ssl-cn**:  Version two with SSL connection and common name information.
+
+### Choosing a health check method
+
+Load Balancers should only forward traffic to “healthy” backend servers. To monitor the health of a backend server, health checks regularly attempt to connect to backend servers using the defined protocol to ensure that servers are up and listening. Various protocols for health checks are available:
+
+* **HTTP**: When selecting HTTP heath checks, you are prompted to choose a **method** (`GET`, `POST` or `PUT`), the **URI** to be targeted, and the response **code** that signals a successful health check.
+* **HTTPS**: The same method and fields as for an HTTP health check apply, but data transmission is encrypted. If you chose HTTPS as the main protocol for the Load Balancer's backend, the Verify Certificate setting you chose there is inherited for health checks. If you chose a different protocol for the Load Balancer's backend, the backend server's certificate will **not** be verified during health checks. See the [dedicated Verify Certificate section](#choosing-a-verify-certificate-setting) for more information.
+* **MYSQL**: For MYSQL health checks, you are prompted to select the database username.
+* **PGSQL**: For PGSQL health checks, you are prompted to select the database username.
+* **LDAP**: No additional settings are required fr LDAP health checks.
```

Comment:
Description of LDAP check available here http://docs.haproxy.org/2.6/configuration.html#4.2-option%20ldap-check 
Extract:

> It is possible to test that the server correctly talks LDAPv3 instead of just
> testing that it accepts the TCP connection. When this option is set, an
> LDAPv3 anonymous simple bind message is sent to the server, and the response
> is analyzed to find an LDAPv3 bind response message.
> 
> The server is considered valid only when the LDAP response contains success
> resultCode (http://tools.ietf.org/html/rfc4511#section-4.1.9).
>Configure the LDAP servers accordingly to allow anonymous binding. You can do this with an IP alias on the server side that allows only HAProxy Enterprise IP addresses to bind to it.

Reply:
For this, and the others, I've added the links to the haproxy doc, it seems to me it'll be cleaner/more useful for the user? Let me know if that's OK with you, or if you'd rather we paraphrased directly into our doc.

Reply:
Actually, I don't think we should link to HAProxy documentation because we are not stating we use HAProxy. We should maybe have a 1 sentence description of the health check (not copying the HAProxy line exactly) and maybe we will create a dedicated page later that will provide more info. In the case of LDAP we could do something like this:
"LDAP health check is testing connection to the LDAP server by sending an a LDAPv3 anonymous simple bind message. This checks the LDAP service is up, and not just that a TCP session can be established."